src/main/java/io/fabric8/maven/docker/StartMojo.java

```diff
@@ -291,9 +316,11 @@ public StartedContainer call() throws Exception {
             //String imageName = new ImageName(imageConfig.getName()).getFullNameWithTag(registry);
 
             String imageName = imageConfig.getName();
-            checkImageWithAutoPull(hub, imageName,
+
+            BuildService.BuildContext buildContext = getBuildContext();
```

Comment:
I see, that's the reason why `getBuildContext()` is in the AbstractDockerMojo.

However, I'm not sure if we would better move `checkImageWithAutoPull()` out of the BuildService. Doesn't sound very "build-y" and there is probably a better place for it, so that the `StartMojo` and the `BuildMojo` could both use it ?